lib/pg_repack.sql.in

```diff
@@ -43,6 +43,37 @@ CREATE FUNCTION repack.get_order_by(oid, oid) RETURNS text AS
 'MODULE_PATHNAME', 'repack_get_order_by'
 LANGUAGE C STABLE STRICT;
 
+CREATE FUNCTION repack.create_log_table(oid) RETURNS void AS
+$$
+BEGIN
+    EXECUTE 'CREATE TABLE repack.log_' || $1 ||
+            ' (id bigserial PRIMARY KEY,' ||
+            ' pk repack.pk_' || $1 || ',' ||
+            ' row ' || repack.oid2text($1) || ')';
+END
+$$
+LANGUAGE plpgsql;
+
+CREATE FUNCTION repack.create_table(oid, name) RETURNS void AS
+$$
+BEGIN
+    EXECUTE 'CREATE TABLE repack.table_' || $1 ||
```

Comment:
Having looked at all the other code, it seems that the prevailing style is `||`, so I'd say let's leave it this way and not consider changing the style unless doing a global refactoring.

Reply:
Indeed, `format` would be much more readable instead of using `||`. I was thinking about using it but I kept `||` because other functions use string concatenation.

> Having looked at all the other code, it seems that the prevailing style is ||, so I'd say let's leave it this way and not consider changing the style unless doing a global refactoring.

I agree, such refactoring would improve readability.

Reply:
I created a ticket for this https://github.com/reorg/pg_repack/issues/369